api/v1alpha1/ngrok_common.go

```diff
@@ -71,3 +73,25 @@ type EndpointWebhookVerification struct {
 	// requests from the given provider. All providers except AWS SNS require a secret
 	SecretRef *SecretKeyRef `json:"secret,omitempty"`
 }
+
+type EndpointCircuitBreaker struct {
```

Comment:
curiuos if these module configurations should be in the moduleset file now that there is a crd they relate to instead of the common file?

Reply:
its not important, i was just wondering if is a remnant of before there was a specific CRD, but since its used in multiple, that makes more sense